NonsoAmadi10/Auto-Mart
# #166766619 Creates API endpoint to update Car Status

### What does this PR do?
- Creates ` PATCH /api/v1/car/:carId/price `

### Description of Task Taken to Complete??
-  Uncomment Test Cases
- Implement feature
- Protect route with JSON webtoken

### How should this be manually Tested??
- Clone this Repo
- ` git checkout ft-user-update-car-status-166766619 `
- Run ` npm install ` to install all dependencies
- Setup all environment variables in the .env.example file
- Run ` npm test ` to see all test cases pass
- Run ` npm run dev-start ` to start the server
- Open postman on ` PATCH /api/v1/car/:carId/status `
- Pass in an integer in the URL path as a valid car id
- Pass in the following in the JSON request body:
>{
> status: FLOAT
> }

### Any Background Context you want to provide??
- You must pass in a token in the request header to access this route. see #50 and #51
- You must own a car advert. see #52 to create one

### What are the relevant pivotal tracker stories??
- [166766619](https://www.pivotaltracker.com/story/show/166766619)
